com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/HooksProcessor.java

```diff
@@ -0,0 +1,38 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.inject.Named;
+
+import com.sap.cloud.lm.sl.cf.process.steps.StepPhase;
+import com.sap.cloud.lm.sl.mta.model.Hook;
+import com.sap.cloud.lm.sl.mta.model.Module;
+
+@Named
+public class HooksProcessor {
```

Comment:
I would say that this is HooksExecutor class as there are only execute* methods in it.